controllers/discovery/searchNews.js

```diff
@@ -40,7 +41,10 @@ const Search = async (req, res) => {
           {
             model: DomainPage,
             as: 'newsLinkDomain',
-            attributes: ['domain_name', 'logo', 'credder_score', 'credder_last_checked']
+            attributes: ['domain_name', 'logo', 'credder_score', 'credder_last_checked'],
+            where: {
+              credder_score: {[Op.gte]: CREDDER_MIN_SCORE}
+            }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new filter on `credder_score` is a good addition for filtering out news links with low scores. However, it might be beneficial to consider the case when `CREDDER_MIN_SCORE` is not defined or null. In such cases, this could lead to unexpected behavior. You should add a check to ensure that `CREDDER_MIN_SCORE` is defined before using it in the query.

```diff
+            where: CREDDER_MIN_SCORE ? {
+              credder_score: {[Op.gte]: CREDDER_MIN_SCORE}
+            } : {}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
this is defined as constant, so should not be null